src/cbc_sdk/endpoint_standard/base.py

```diff
@@ -617,6 +620,15 @@ def timeout(self, msecs):
         self._timeout = msecs
         return self
 
+    def get_details(self):
```

Comment:
This should not be on the EnrichedEventQuery instead this should be on the EnrichedEvent model

Reply:
The current implementation will overload the Details API when an EnrichedEventQuery matches more than 100 events. The best practice indicated by the Nic Cage team also recommends we fetch the details for a singular event as that is more optimal for the backend.

100 limit is documented https://developer.carbonblack.com/reference/carbon-black-cloud/platform/latest/platform-search-api-enriched-events/#request-details-for-enriched-events-v2

Reply:
Done.